### PR TITLE
remove unused envs

### PR DESCRIPTION
**Pull request by deepikasl**
https://github.com/Shippable/base/issues/892

### CODE REVIEW

File: versions/v5.1.2.json [hunk 1]
```diff
@@ -271,30 +271,25 @@
       "isGlobal": true,
```
Comment (shrivara) [inline versions/v5.1.2.json]: How will `forever` work this change? If you look at `boot.sh` it makes use of RUN_MODE env.

Comment (deepikasl) [inline versions/v5.1.2.json]: for www: https://github.com/Shippable/www/blob/master/base#L63 we are defining in base.
for api: https://github.com/Shippable/api/blob/master/base#L63

Comment (shrivara) [inline versions/v5.1.2.json]: We should do it for other repos too. This will break local dev otherwise.

Comment (shrivara) [inline versions/v5.1.2.json]: RUN_MODE should be removed from logup as well.

**Pull request merged**